src/model/core.py

```diff
@@ -95,15 +102,19 @@ def add_images(ims, value=None):
     im_sum: 2d/3d numpy array
     """
     im_sum = ims[0].astype(float)
```

Comment:
I was wondering reading thos functions, why do you force operations to be made with float array?

Reply:
if I add a value to an image of dtype int8, the output image will be also of dtype int8.
Then for one pixel if I have 253 + 5 the result will be 3, not 258. 
Maybe I can work around this issue with numpy.add, will check that

Reply:
[#36] Lets talk about that on this issue s'il vous plait